weldx/tests/asdf_tests/test_weldx_file.py

```diff
@@ -593,9 +593,6 @@ def test_cannot_update_del_protected_keys(self, protected_key):
         """Ensure we cannot manipulate protected keys."""
         expected_match = "manipulate an ASDF internal structure"
         warning_type = UserWarning
-        # try to obtain key from underlying dict.
-        with pytest.raises(KeyError), pytest.warns(WeldxDeprecationWarning):
-            _ = self.fh.data[protected_key]
```

Comment:
can this be deleted now? @marscher

Reply:
For some reason the first line below "changes" in the RTD version of the changelog is shown in bold letters: 

https://weldx--744.org.readthedocs.build/en/744/changelog_link.html

Checked the changelog file and the only difference to the other lines that I could find is that the line in question has only a single space between the `-` and the text while all others have 2 spaces. I am not that familiar with RST, but I guess that's the problem.

Reply:
good find @vhirtham , let's see if it is fixed with the spaces (otherwise we can reformat later)

Reply:
@marscher argh... this should have been an extra comment ... ooops. Well lets see what fixes it